library/Icinga/Crypt/AesCrypt.php

```diff
@@ -76,6 +79,28 @@ public function setMethod($method)
         return $this;
     }
 
+    /**
+     * Get supported method
+     *
+     * @return string
+     */
+    public function getSupportedMethod()
```

Comment:
I'd rather make this protected. At least, I see no need why it should be of use externally?

Reply:
Yes, you are right. Fixed